frameit/lib/frameit/screenshot.rb

```diff
@@ -88,8 +88,6 @@ def frame_orientation
         orientation = block.call(filename)
         valid = [:landscape_left, :landscape_right, :portrait, nil]
         UI.user_error("orientation_block must return #{valid[0..-2].join(', ')} or nil") unless valid.include?(orientation)
-
-        puts("Forced orientation: #{orientation}")
```

Comment:
Strictly speaking unrelated change. Is this ok @milch?

Reply:
Yeah sure, this can be removed if you think it makes sense

Reply:
I see 5 of these lines in my Terminal per screenshot where `orientation` is nil. So instead of removing it, it can also be extended with an if condition:

```puts("Forced orientation: #{orientation}") unless orientation == nil```

Reply:
You added this out the `unless`, which should be fine.